src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/InputIndentationDifficultAnnotations.java

```diff
@@ -37,8 +37,8 @@ void foo() { //indent:4 exp:4
 } //indent:0 exp:0
 
 @DifficultAnnotation({ //indent:0 exp:0
-@MyType(value = Boolean.class, name = "boolean"), //indent:0 exp:4 warn
-@MyType(value = String.class, name = "string") }) //indent:0 exp:4 warn
+@MyType(value = Boolean.class, name = "boolean"), //indent:0 exp:4,23,25 warn
+@MyType(value = String.class, name = "string") }) //indent:0 exp:4,23,25 warn
```

Comment:
`23,25`
I do not have a good knowledge of this check, can you shortly describe why there are such big numbers for expected indentation?

Reply:
The children are accepting indent levels like arrayInitIndent, lineWrapping and basicOffset.
Those big numbers are because it is also accepting indentLevel from firstNonBlank line after lcurly (Initialization block's) inside which those children are placed.

`23, 25`:
because indentation comment is the first non blank line on after the lcurly. And it is placed at 23 indent level. so 23 and 23+arrayInitIndent(2) = 25.

This is important/helpful when we have code style like this: (where we are indenting other children relative to previous ones) 
```
@Annotation({ @Child1,
              @Child2, 
              @Child3
})
``` 

So yes, if there would not have those comments there then we would not have got `23,25`.